CHANGELOG.md

```diff
@@ -106,7 +108,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#1824](https://github.com/nix-rust/nix/pull/1824))
 - Workaround XNU bug causing netmasks returned by `getifaddrs` to misbehave.
   ([#1788](https://github.com/nix-rust/nix/pull/1788))
-  
```

Comment:
I believe you have your editor set to automatically delete trailing whitespaces. I'm not against it, but that should be done as a separate clean up commit.